### PR TITLE
Template and module updating

### PR DESCRIPTION
**Pull request by gq1**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/variantcalling/tree/main/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by gq1** [comment]
@sanger-tolsoft fix linting

**Comment by gq1** [comment]
@nf-core-bot fix linting

**Comment by sanger-tolsoft** [comment]
@nf-core-bot fix linting

**Comment by sanger-tolsoft** [comment]
@nf-core-bot fix linting

**Review by priyanka-surana — DISMISSED** [review]
One minor change. Otherwise all good. 👍

**Comment by priyanka-surana** [comment]
The singularity tests are failing for deepvariant. This is not the case in modules repo, so it is some specific to our pipeline.
```
Pulling Singularity image docker://nf-core/deepvariant:1.5.0 [cache /lustre/scratch123/tol/teams/tolit/users/ps22/bin/singularity/nf-core-deepvariant-1.5.0.img]
ERROR ~ Error executing process > 'SANGERTOL_VARIANTCALLING:VARIANTCALLING:DEEPVARIANT_CALLER:DEEPVARIANT (3)'

Caused by:
  Failed to pull singularity image
  command: singularity pull  --name nf-core-deepvariant-1.5.0.img.pulling.1695729700476 docker://nf-core/deepvariant:1.5.0 > /dev/null
  status : 255
  message:
    FATAL:   While making image from oci registry: error fetching image to cache: failed to get checksum for docker://nf-core/deepvariant:1.5.0: reading manifest 1.5.0 in docker.io/nf-core/deepvariant: errors:
    denied: requested access to the resource is denied
    unauthorized: authentication required
```

File: nextflow.config [hunk 1]
```diff
@@ -89,46 +96,68 @@ profiles {
...
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     arm {
         docker.runOptions = '-u $(id -u):$(id -g) --platform=linux/amd64'
     }
     singularity {
         singularity.enabled    = true
         singularity.autoMounts = true
+        conda.enabled          = false
```
Comment (priyanka-surana) [inline nextflow.config:118]:
```suggestion
        singularity.registry   = 'quay.io'
        conda.enabled          = false
```

**Review by priyanka-surana — APPROVED** [review]
All tests working. Yay 💃🏼

**Pull request merged**